operator_ui/@types/core/store/models.d.ts

```diff
@@ -625,11 +625,15 @@ declare module 'core/store/models' {
     }
   }
 
-  export type LogConfigLevel = 'debug' | 'info' | 'warn' | 'error'
+  // We really need to change the API for this. It not only returns levels but
+  // true/false for IsSQLEnabled
+  export type LogConfigLevel = 'debug' | 'info' | 'warn' | 'error' | "true" | "false"
+  export type LogServiceName = 'Global' | 'IsSqlEnabled' | 'header_tracker' | 'fluxmonitor'
 
   export interface LogConfig {
```

Comment:
This is a horrible API but I there are changes being made around logging in the node and I don't want to change it just yet. 

Essentially there are 2 arrays one containing the key (called ServiceName) and values (Called LogLevelConfig). The index of the array maps each key to the value in their respective arrays.